### PR TITLE
update the base image to 2110

### PR DESCRIPTION
**Pull request by wyli**
### Description
this update will fix the test error:
https://github.com/Project-MONAI/MONAI/runs/4158407949?check_suite_focus=true

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by wyli** [comment]
tested with https://github.com/Project-MONAI/MONAI/runs/4158569724?check_suite_focus=true

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I think the root cause may related to this PyTorch PR: https://github.com/pytorch/pytorch/pull/66191?

Thanks.

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> I think the root cause may related to this PyTorch PR: [pytorch/pytorch#66191](https://github.com/pytorch/pytorch/pull/66191)?
> 
> Thanks.

yes, looks like that's the issue

**Comment by Nic-Ma** [comment]
> > Hi @wyli ,
> > I think the root cause may related to this PyTorch PR: [pytorch/pytorch#66191](https://github.com/pytorch/pytorch/pull/66191)?
> > Thanks.
> 
> yes, looks like that's the issue

As this is an obvious bug in PyTorch, maybe we can skip the test for this version directly?

Thanks.

**Comment by wyli** [comment]
pytorch 21.10 doesn't have this issue, as confirmed by Vish. that's why I'm making this PR.

**Comment by wyli** [comment]
Any concern about this PR? @Nic-Ma this is currently blocking the constructive loss function.

**Review by Nic-Ma — APPROVED** [review]

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I don't quite understand what code change in this PR fixed the `F.cosine_similarity` issue mentioned in the description? But anyway, the PR updated docker to 21.10, looks good to me.

Thanks.
